core/src/main/scala/org/apache/spark/scheduler/TaskSetBlacklist.scala

```diff
@@ -61,6 +61,8 @@ private[scheduler] class TaskSetBlacklist(val conf: SparkConf, val stageId: Int,
   private val blacklistedExecs = new HashSet[String]()
   private val blacklistedNodes = new HashSet[String]()
 
+  var taskSetLatestFailureReason: String = null
```

Comment:
Can we please avoid public variables here? Also why not make it less verbose to change to `latestFailureReason`?

Reply:
I have thought about `public` problem,but we need to get this value from `TaskSetManager`. If i add a `def getLatestFailureReason` make sense? @jerryshao